retriever/__main__.py

```diff
@@ -196,6 +196,8 @@ def main():
             for dataset in scripts:
                 print("=> Installing", dataset.name)
                 try:
+                    if not os.path.exists(args.data_dir):
```

Comment:
So we looked at the logic and this works fine. we added it to the PR https://github.com/harshitbansal05/retriever/pull/1/files#diff-ff05cd5f49d751e66a61636b498c70acR730, So incase you @ashishpriyadarshiCIC and @harshitbansal05 collaborate, we should have one PR that is clean and has, may be two commits or one commit with two people.

Look up how to make a commit with two contributors:
One author commits  but adds the author name and email of the other. using --author.

Reply:
Ok Sir